pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -255,7 +256,17 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		}
 
 		if len(vpcSubnets) == 0 {
-			vpcSubnets = append(vpcSubnets, fmt.Sprintf("vpc-subnet-%s", clusterID.InfraID))
+			if capiutils.IsEnabled(installConfig) {
+				for i := 1; i <= 3; i++ {
```

Comment:
It would be nice to add a comment about where the `3` is coming from.

Reply:
Done